perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/SecurityTeamsManagerBlImpl.java

```diff
@@ -130,10 +130,25 @@ public SecurityTeam getSecurityTeamByName(PerunSession sess, String name) throws
 	}
 
 	@Override
-	public List<User> getAdmins(PerunSession sess, SecurityTeam securityTeam) throws InternalErrorException {
-		return getSecurityTeamsManagerImpl().getAdmins(sess, securityTeam);
+	public List<User> getAdmins(PerunSession sess, SecurityTeam securityTeam, boolean onlyDirectAdmins) throws InternalErrorException {
+		if(onlyDirectAdmins) {
+			return getSecurityTeamsManagerImpl().getDirectAdmins(sess, securityTeam);
+		} else {
+			return getSecurityTeamsManagerImpl().getAdmins(sess, securityTeam);
+		}
 	}
 
+        @Deprecated
+        @Override
+        public List<User> getDirectAdmins(PerunSession sess, SecurityTeam securityTeam) throws InternalErrorException {
```

Comment:
I don't think is good to create "deprecated" method from start. I think that getAdmins with "onlyDirectAdmins" parameter is enough.